### ISSUE TITLE
Cannot build project

### ISSUE THREAD

**Issue by phoenixrising1800** [atomice1/bluecheese#1]
When trying to build on MacOS Monterey 12.5 running `cmake .`, Cmake throws an error at CMakeLists.txt:28:

`CMake Error at CMakeLists.txt:28 (find_package):
  Could not find a package configuration file provided by "QT" with any of
  the following names:

    Qt6Config.cmake
    qt6-config.cmake

  Add the installation prefix of "QT" to CMAKE_PREFIX_PATH or set "QT_DIR" to
  a directory containing one of the above files.  If "QT" provides a separate
  development package or SDK, be sure it has been installed.`

**Comment by atomice1** [comment]
Hi! Bluecheese requires Qt SDK 6.6 or later to build (sorry, I should add that to the README!) If you already have the Qt SDK installed: Which version? Did you use the Qt installer, Homebrew, or some other method?

**Comment by phoenixrising1800** [comment]
Thanks for the quick reply; I was able to install Qt on my system via Homebrew and ran _make_ to build the project. 
However, I ran into a problem in the `src > desktop` folder generated: I am unable to open/run the "bluecheese-gui.app" desktop application with the error: "**You have macOS 12.5. The application requires macOS 13.0 or later.**" Otherwise, project is neat, seems functional and I might try to run cli option.
